cmd/run.go

```diff
@@ -243,7 +243,7 @@ func init() {
 	RunCmd.Flags().StringVarP(&logLevel, "log-level", "", "info", "Sets the log verbosity. Valid values are: debug, info, warn, error, fatal, or panic. Default is info")
 	RunCmd.Flags().IntVarP(&maxConcurrency, "max-concurrency", "", -1, "controls the concurrency level of the app. Default is unlimited")
 	RunCmd.Flags().StringVarP(&protocol, "protocol", "", "http", "tells Dapr to use HTTP or gRPC to talk to the app. Default is http")
-	RunCmd.Flags().StringVarP(&componentsPath, "components-path", "", standalone.GetDefaultComponentsFolder(), "Path for components directory. Default is ~/.dapr/components or %USERPROFILE%\\.dapr\\components")
+	RunCmd.Flags().StringVarP(&componentsPath, "components-path", "", standalone.GetDefaultFolderPath(standalone.DefaultComponentsDirName), "Path for components directory. Default is ~/.dapr/components or %USERPROFILE%\\.dapr\\components")
```

Comment:
could you update the default here to be $HOME/.dapr/config as it covers Linux also